README.md

```diff
@@ -7,6 +7,7 @@ An Ansible role for installing Open Trip Planner
 - `otp_data_dir` - Local directory to store OTP data (default: `/var/otp`)
 - `otp_repo` - Git repo to pull source from (default: `https://github.com/opentripplanner/OpenTripPlanner`)
 - `otp_version` - Commit to pull from (default: `339c1deb1daaa5edb63cbcd738318591ad9a61c7`, 0.11.x branch)
+- `otp_jarfile` - JAR file to run in daemon, absolute, or relative to `otp_bin_dir` (default: `./otp-core/target/otp.jar`, 0.11.x branch)
```

Comment:
The only reason I added the absolute wording there is because it would work with the way the variable is pasted into the upstart script. Might be best to just change the README text to be:

JAR file to run in daemon, or relative to `otp_bin_dir`

since OTP always builds the jar inside the bin dir.